yb-voyager/cmd/cutover.go

```diff
@@ -27,7 +27,7 @@ import (
 
 var cutoverCmd = &cobra.Command{
 	Use:   "cutover",
-	Short: "Prepare to point your application to a different database during live migration.",
+	Short: "Pointing your application to a different database during live migration.",
```

Comment:
Consider reverting this particular change.

Reply:
I made this change as the top-level `cutover` command is only for `cutover status` and is no more a prepare thing as that is handled in `initiate cutover`. Let me know what you think about this.

Reply:
@shubham-yb I think cutover help is fine, the thing you are talking about is for `cutover status` but the usage of cutover should remain this only. Reverted this change.